src/components/MapFacetContents.js

```diff
@@ -200,14 +200,24 @@ class MapFacetContents extends React.Component<MapFacetContentsDefaultProps, Map
 
       const points = this.props.buckets.map((bucket) => {
         const value = bucket.value; // JSON.parse(bucket.value);
+        let longitude = 0;
+        let latitude = 0;
+        if (typeof value === 'string') {
+          const valueArr = value.split(',');
+          longitude = valueArr[0];
```

Comment:
Also, you should make sure that there are 2 items in the array you get from `split()`... if there's some random string without a comma, you'll get an error trying to access `valueArr[1]`.